### PR TITLE
Scottx611x/analysis custom name bugfix

### LINKED ISSUE #1893: Naming analyses doesn't seem to do anything?
Giving a non-default names to a new analysis  doesn't seem to do anything, as far as I can tell: Were would I expect to see it be used?

### PR DESCRIPTION
**Pull request by scottx611x**
Fixes #1893
- Bugfix for user-specified Analysis names not being assigned
- Add test coverage to ensure that user-specified names are properly set

### CODE REVIEW

File: refinery/analysis_manager/utils.py [hunk 1]
```diff
@@ -194,19 +163,17 @@ def create_noderelationship_analysis(validated_analysis_config):
 
     # ANALYSIS MODEL
     # How to create a simple analysis object
-    if not custom_name:
-        temp_name = "{} {}".format(
+    if not name:
+        name = "{} {}".format(
```
Comment (mccalluc) [inline refinery/analysis_manager/utils.py]: Is there logic that should be shared between here and line 81? (This feels like copy-and-paste which preceded this commit, so not a high priority to fix.)

File: refinery/analysis_manager/utils.py [hunk 2]
```diff
@@ -413,3 +380,75 @@ def validate_analysis_config(analysis_config):
...
+        )
+
+
+def _fetch_solr_uuids(nodeset_instance):
+    """
+    Fetches a NodeSet instance from a given UUID
+    :param node_set_uuid: UUID String
+    :return: <NodeSet>
+    :raises: RuntimeError
+    """
```
Comment (mccalluc) [inline refinery/analysis_manager/utils.py]: Docs seem to be copy and paste, but don't match this method's signature?

Comment (scottx611x) [inline refinery/analysis_manager/utils.py]: Nice find, thanks :)

File: refinery/analysis_manager/utils.py (hunk 1 above)
Comment (scottx611x) [inline refinery/analysis_manager/utils.py]: Yes definitely, I can refactor

**Pull request merged**